src/components/Dashboard.vue

```diff
@@ -39,7 +39,7 @@
                     <v-icon color="teal">schedule</v-icon>
                   </v-list-tile-action>
                   <v-list-tile-content>
-                    <v-list-tile-title>{{ chargingTimeLeft}} h</v-list-tile-title>
+                    <v-list-tile-title>{{ chargingTimeLeft }} h / {{ finishTime }}</v-list-tile-title>
```

Comment:
Would be better, if time is in brackets

00:37 h (14:38) for example

Reply:
done this